roles/platform/tasks/main.yml

```diff
@@ -74,6 +74,12 @@
       ansible.builtin.include_tasks:
         file: install-platform.yml
 
+- name: Create systemd file
```

Comment:
If this was just added, as the diff indicates, how was anything working before?  I'm confused...

Should we add owner and group?

Reply:
I added the owner/group.